src/components/UI/Backdrop/Backdrop.js

```diff
@@ -4,16 +4,16 @@ import { CSSTransition } from 'react-transition-group';
 import './Backdrop.scss';
 
 const Backdrop = (props) => {
-  const { isModalOpen, close } = props;
+  const { isOpen, type, close } = props;
```

Comment:
You could either default `type` here as follows
```suggestion
  const { isOpen, type = 'default', close } = props;
```

or utilize the React `.defaultProps` API to make sure that you don't get a weird className

Reply:
ahh ok good idea. That's the first time I'm using that!